avaframe/tests/test_cfgHandling.py

```diff
@@ -0,0 +1,185 @@
+
+    simNames = cfgHandling.filterSims(avaDir2, parametersDict, specDir=avaDir2)
+    simNames = sorted(simNames)
+
+    assert len(simNames) == 2
+    assert simNames == ['relGar_6f35cbd808_null_dfa', 'relGar_b9b17dd019_ent_dfa']
+
+    parametersDict = {'relTh': [1., 1.5]}
+
+    simNames = cfgHandling.filterSims(avaDir2, parametersDict, specDir=avaDir2)
+    simNames = sorted(simNames)
+
+    assert len(simNames) == 4
+    assert simNames == ['relGar_1022880a70_null_dfa', 'relGar_6f35cbd808_null_dfa',
+                        'relGar_b9b17dd019_ent_dfa', 'relGar_c4337e50ac_null_dfa']
+
+
+def test_applyCfgOverride():
+    """ test overriding cfg parameters in a cfg object from another cfg with an override section """
+
+    cfgWithOverrideParameters = configparser.ConfigParser()
+    cfgWithOverrideParameters['GENERAL'] = {'testp1': 1., 'testp2': 'testValue'}
+    cfgWithOverrideParameters['com1DFA_override'] = {'defaultConfig': True, 'mu': 0.7, 'tStep': 100., 'notParameter': 1.}
+
+    cfgToOverride = configparser.ConfigParser()
+    cfgToOverride['GENERAL'] = {'mu': 1., 'tEnd': 400., 'relTh': 1.}
+    cfgToOverride['DFA'] = {'tStep': 0.3, 'dt1': 6., 'flowF': 3.}
+
+    cfgToOverride, cfgWithOverrideParameters = cfgHandling.applyCfgOverride(cfgToOverride, cfgWithOverrideParameters, com1DFA, addModValues=False)
+
+    assert cfgToOverride['GENERAL']['mu'] == '0.7'
+    assert cfgToOverride['GENERAL']['tEnd'] == '400.0'
+    assert cfgToOverride['GENERAL']['relTh'] == '1.0'
+    assert cfgToOverride['DFA']['tStep'] == '100.0'
+    assert cfgToOverride['DFA']['dt1'] == '6.0'
+    assert cfgToOverride['DFA']['flowF'] == '3.0'
+    assert cfgWithOverrideParameters['GENERAL']['testp1'] == '1.0'
+    assert cfgWithOverrideParameters['GENERAL']['testp2'] == 'testValue'
+    assert cfgWithOverrideParameters['com1DFA_override']['defaultConfig'] == 'True'
+    assert cfgWithOverrideParameters['com1DFA_override']['mu'] == '0.7'
+    assert cfgWithOverrideParameters['com1DFA_override']['tStep'] == '100.0'
+    assert cfgWithOverrideParameters['com1DFA_override']['notParameter'] == '1.0'
+    assert len(cfgWithOverrideParameters.items('GENERAL')) == 2
+    assert len(cfgWithOverrideParameters.items('com1DFA_override')) == 4
+    assert len(cfgToOverride.items('GENERAL')) == 3
+    assert len(cfgToOverride.items('GENERAL')) == 3
+
+
+    cfgWithOverrideParameters = configparser.ConfigParser()
+    cfgWithOverrideParameters['GENERAL'] = {'testp1': 1., 'testp2': 'testValue'}
+    cfgWithOverrideParameters['com1DFA_override'] = {'defaultConfig': True, 'mu': 0.7, 'tStep': 100., 'notParameter': 1.}
+
+    cfgToOverride = configparser.ConfigParser()
+    cfgToOverride['GENERAL'] = {'mu': 1., 'tEnd': 400., 'relTh': 1.}
+    cfgToOverride['DFA'] = {'tStep': 0.3, 'dt1': 6., 'flowF': 3.}
+
+    cfgToOverride, cfgWithOverrideParameters = cfgHandling.applyCfgOverride(cfgToOverride, cfgWithOverrideParameters, com1DFA, addModValues=True)
```

Comment:
you could test that the log.warning('Additional Key [\'%s\'] in section %s_override is ignored.' % (item, modName))
works using
with caplog.at_level(logging.WARNING):
        cfgToOverride, cfgWithOverrideParameters = cfgHandling.applyCfgOverride(cfgToOverride,
                                                         cfgWithOverrideParameters, com1DFA, addModValues=True)
    assert (the warning message) in caplog.text

Reply:
added this